templates/careers/support-engineering.html

```diff
@@ -1,88 +1,28 @@
-    <div class="row">
-      <div class="col-6">
-        <h3 class="p-heading--2">Who you are</h3>
-      </div>
-      <div class="col-6">
-        <p>If you enjoy helping customers, solving really complex problems and thrive on operating software or building software, then Support Engineering at Canonical offers a challenging work environment where you can learn and grow.</p>
-      </div>
-    </div>
-  </div>
-</section>
+{% block meta_description %}Canonical's Engagement Project Management Office team owns, drives and communicates delivery excellence by providing strong Project process guidelines, project continuity and data analytics.{% endblock %}
 
-{% include "shared/_social-icons-section.html" %}
-
-{% with section_classes='p-strip', heading_topic='Support Engineering', tag_id='1374', limit='3' %}
-  {% include "shared/_blog-card.html" %}
+{% block content %}
+{% with 
+    department=department,
+    claim="Support Engineering roles include technology experts and support operators across a vast portfolio of open source projects. We support, test, fix and operate the Ubuntu OS, infrastructure such as Kubernetes, LXD, OpenStack, MAAS, and a growing portfolio of open source applications. We believe in customer-centricity, working with the community and enabling smart operations through code.",
+    fast_track_jobs=fast_track_jobs,
+    block_1_subheading='Support Engineering at Canonical',
+    block_1_text=["In Support Engineering, you will enjoy the excitement of a team that is fun, diverse, distributed globally and incredibly competent. Every day, we deliver the expertise our customers need to solve some of the world’s most challenging problems.", "While we all enjoy the feeling that comes from passionately working together, we also collectively respect each other's lives away from work. Part of standing together with a global team that is set up to work remotely means you can step away during your off-hours and know that your team has your back. Your manager will guide and mentor you through your work but equally wants you to make sure you are taking the time away from work to help you unwind and refocus. All of us are in this together, and we want you in Support Engineering for the long term."],
+    image_url='https://assets.ubuntu.com/v1/456c9c0f-department-support-engineering-%402x.jpg',
+    image_width="1500",
+    image_height="564",
+    block_2_heading="Projects you'll work on",
+    block_2_text=["Canonical's project managers are masters at engaging stakeholders. They ensure the success of projects by taking ownership, giving direction and driving communication with stakeholders. They champion respect, integrity and accountability.", "Collaboration underpins everything the team does. Upskilling and prioritising innovation drives our pursuit of excellence. Successful project managers are proactive, skilled at evaluating projects and driving client vision with new business opportunities in mind."],
```

Comment:
```suggestion
    block_2_text=["We build strong relationships with customers and stakeholders. This may involve regular check-ins with key customers, hosting customer events, or performing service reviews.", "We work closely with engineering teams to ensure that customer issues are addressed in a timely manner, and that support requirements are considered during the product development process.", "Our team also improves the quality of support documentation and resources. This may involve updating existing documentation, creating new knowledge base articles, or developing tutorials and training materials."],
```